lib/config/testdata/tpm_ekcert_ca.pem

```diff
@@ -0,0 +1,24 @@
+This is a genuine PEM for STM TPM EK Intermediate CA 06
```

Comment:
Do we have a download URL to add here too?

Reply:
9faeb891fe